schematic/models/validate_attribute.py

```diff
@@ -390,6 +390,73 @@ def generate_content_error(
+        dmge: DataModelGraphExplorer, attribute_name: str, val_rule: str
+    ) -> list[list[str]]:
+        """Raise a warning if no columns were found in the specified project to validate against, inform user the
+        source manifest will be uploaded without running validation. Retain standard warning
+        Args:
+            dmge: DataModelGraphExplorer object
+            attribute_name, str: str, attribute being validated
+            val_rule: str, defined in the schema.
+        Returns:
+            Errors: list[str] Error details for further storage.
+            warnings: list[str] Warning details for further storage.
+        """
+        error_message = (
+            "Cross Manifest Validation Warning: There are no target columns to validate "
+            f"this manifest against for attribute: {attribute_name}, "
+            f"and validation rule: {val_rule}. It is assumed this is the first manifest in a "
+            "series to be submitted, so validation will pass, for now, and will run again "
+            "when there are manifests uploaded to validate against."
+        )
+
+        _, warnings = GenerateError.raise_and_store_message(
+            dmge=dmge,
+            val_rule=val_rule,
+            error_row=None,
+            error_col=attribute_name,
+            error_message=error_message,
+            error_val=None,
+            message_level="warning",
+        )
+
+        return [warnings]
+
+    def generate_no_value_in_manifest_error(
+        dmge: DataModelGraphExplorer, attribute_name: str, val_rule: str
+    ) -> tuple[list[str], list[str]]:
+        """Raise a warning or error based on the messaging level, if target manifests have been found to
+        validate against but the manifest itself does not contain data (across the entire manifest,
+        not just the column being validated.)
+        Args:
+            dmge: DataModelGraphExplorer object
+            attribute_name, str: str, attribute being validated
+            val_rule: str, defined in the schema.
+        Returns:
+            Errors: list[str] Error details for further storage.
+            warnings: list[str] Warning details for further storage.
+        """
+        errors, warnings = [], []
+        error_message = (
+            f"Cross Manifest Validation: There were manifests found to match for attribute: {attribute_name} "
+            f"and validation rule: {val_rule}, but no data was found in this manifest to validate against."
+        )
+
+        nv_errors, nv_warnings = GenerateError.raise_and_store_message(
+            dmge=dmge,
+            val_rule=val_rule,
+            error_row=None,
+            error_col=attribute_name,
+            error_message=error_message,
+            error_val="No Invalid Entry Recorded",
+        )
+        if nv_errors:
+            errors.append(nv_errors)
+        if nv_warnings:
+            warnings.append(nv_warnings)
+        return errors, warnings
```

Comment:
I don't understand why the return isn't just:
`return nv_errors, nv_warnings`
?